core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -736,7 +742,7 @@ var ExplorationEditorMainTab = function() {
           'Parameter Element Button', parameterElementButton);
 
         var multipleChoiceAnswerOption =
-          await multipleChoiceAnswerOptions(parameterValues[i]);
+          await multipleChoiceAnswerOptions(parameterValues[i])[0];
```

Comment:
Will an undefined case not exist here?

Reply:
No, as the element is present everytime we call the selector.